docs/appendix.rst

```diff
@@ -200,6 +200,24 @@ This documentation uses a few special terms to refer to Python types:
       define the use of a query language, it is specified with the string
       ``DMTF:QCL``. CQL is described in DMTF standard :term:`DSP0202`.
 
+   Schema
+      A schema (named CIM schema in the DMTF specification :term:`DSP0004`) in
```

Comment:
In DSP0004 2.8.0, the same terms are used:
* "schema: a set of classes with a single defining authority or owning organization"
* "CIM schema: the schema published by the DMTF that defines the Common Information Model"
Given that, I think we can use these definitions pretty much with the same wording.

Reply:
Changed to use everywhere schema and CIM schema.